sklearn/linear_model/stochastic_gradient.py

```diff
@@ -700,17 +765,42 @@ class SGDClassifier(BaseSGDClassifier):
         - 'constant': eta = eta0
         - 'optimal': eta = 1.0 / (alpha * (t + t0)) [default]
         - 'invscaling': eta = eta0 / pow(t, power_t)
+        - 'adaptive': eta = eta0, as long as the training keeps decreasing.
+        Each time n_iter_no_change consecutive epochs fail to decrease the
```

Comment:
I don't think this will format right. You need to indent to be under the bullet point. Another option is to reformat this as a definition list (with term on one line and definition indented on the next; no bullets).